examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1575 @@
+{
```

Comment:
<p>There is also important flag <code>keep_level_columns</code> which might be meaningful for exog  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>